packages/typegpu/src/experimental/index.ts

```diff
@@ -29,8 +35,12 @@ export const tgpu = {
   vertexLayout,
   bindGroupLayout,
 
-  init,
-  initFromDevice,
+  init: init as (
+    options?: InitOptions | undefined,
+  ) => Promise<ExperimentalTgpuRoot>,
+  initFromDevice: initFromDevice as (
+    options: InitFromDeviceOptions,
+  ) => ExperimentalTgpuRoot,
```

Comment:
what's the difference?

Reply:
Forgot to change the default `init` and `initFromDevice` 🤦. It should make more sense now. We were exporting `init` and `initFromDevice` as returning `ExperimentalTgpuRoot` from the non-experimental entry point.